README.md

```diff
@@ -15,12 +15,21 @@ This module deploys the following observability agents to a Red Hat OpenShift Co
 <!-- BEGIN OVERVIEW HOOK -->
 ## Overview
 * [terraform-ibm-observability-agents](#terraform-ibm-observability-agents)
+* [Submodules](./modules)
+    * [logs-agent-module](./modules/logs-agent-module)
 * [Examples](./examples)
-    * [Deploy basic observability agents](./examples/basic)
+    * [Log Analysis agent](./examples/basic)
+    * [Monitoring agent + Cloud Logs agent on Kubernetes using CSE ingress endpoint with an apikey](./examples/logs-agent-iks)
+    * [Monitoring agent + Cloud Logs agent on OCP using VPE ingress endpoint with a Trusted Profile](./examples/logs-agent-roks)
 * [Contributing](#contributing)
 <!-- END OVERVIEW HOOK -->
 
 ## terraform-ibm-observability-agents
+
+### Deprecated: Log Analysis
+
+**Important:** IBM Log Analysis will be discontinued on 30 March 2025 and replaced by IBM Cloud Logs.
```

Comment:
Can we use same format / terminology as https://github.com/terraform-ibm-modules/terraform-ibm-observability-instances/blob/main/README.md ?

So maybe go with:

> [!IMPORTANT]
> The IBM Log Analysis service is deprecated. [IBM Cloud Logs](https://www.ibm.com/products/cloud-logs) is the replacement service and is now the agent that is deployed by this module.